addons/anonymization/wizard/anonymize_wizard.py

```diff
@@ -266,10 +266,10 @@ def reverse_anonymize_database(self):
         # load the json/pickle file content into a data structure:
         content = base64.decodestring(self.file_import)
         try:
-            data = json.loads(content)
+            data = json.loads(content.encode('utf8'))
```

Comment:
That doesn't look correct. base64.decodestring should return bytes, json.loads takes text. So `content` should be *decoded* before being tentatively json-loaded, not re-encoded.